.github/workflows/docker.yml

```diff
@@ -0,0 +1,47 @@
+name: docker_build
+
+on:
+  push:
+    branches:
+      - 'dev'
```

Comment:
Will this line trigger the workflow at every push to the `dev` branch for every user that has forked TTK?

Reply:
Yes, that's right. Didn't think about the forks.
@julien-tierny What should I do about this?

Reply:
What I do usually for the other workflows is do disable the branch push trigger and enable them only for pull requests (maybe not relevant in this case) but also enable the push trigger on specific tags (with at least one unique tag per workflow).

See for instance .github/workflows/package.yml (or every other workflow) for how I've done it.

The README file in the workflows directory also provides some user help on how to create and push tags to trigger the specific workflows.

Reply:
Ah perfect, thanks!
So how should we proceed? Should I do another PR for that?